Dockerfile

```
@@ -0,0 +1,34 @@
+FROM python:buster as base
+
+# For Production
+
+FROM base as production
+
+RUN apt-get update && apt-get install -y curl
+RUN curl -sSL https://install.python-poetry.org | /usr/bin/python3 -
+RUN mkdir -p /app/todo_app
+COPY poetry* /app
+COPY pyproject.toml /app
+COPY todo_app /app/todo_app
```

We shouldn't need to create the directories before copying, but if we remove that line here then it does error; the reason for that is by default it thinks you're copying the `pyproject.toml` file into a _file_ called `app`. We can fix this a few ways, but one nice one might be:
```suggestion
WORKDIR /app # creates the directory for us if missing
COPY poetry* .
COPY pyproject.toml .
COPY todo_app ./todo_app
```